README.md

```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
 
 For information regarding updating Wrangler, click [here](https://workers.cloudflare.com/docs/quickstart/updating-the-cli/).
 
+## Getting Started
+
+Once you have installed Wrangler, spinning up and deploying your first Worker is easy!
+
+```console
+$ wrangler generate my-worker
+$ cd my-worker
+# update your wrangler.toml with your Cloudflare Account ID
+$ wrangler config # => enter your api key and email
```

Comment:
```suggestion
$ wrangler config 
```
This is changing so I say just remove and let the prompt of the command instruct them